### PR TITLE
Fix Lightbox prev / next on feed

### PR DESCRIPTION
**Pull request by broskoski**
Closes https://github.com/aredotna/ervell/issues/1693

This doesn't automatically load the next page when the user reaches the end (almost), but at least next / previous work now.

### CODE REVIEW

File: react/util/updateMediatorBlocks.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import Blocks from 'collections/blocks.coffee';
+
+export default ({ blocks }) => {
+  mediator.shared.blocks = new Blocks(blocks.map(k => ({
+    id: k.id,
+    class: k.__typename,
+    base_class: 'Block',
+    unloaded: true,
+  })));
+};
```
Comment (dzucconi) [inline react/util/updateMediatorBlocks.js]: Just wondering why this is needed?

File: react/components/Feed/index.js [hunk 2]
```diff
@@ -33,6 +35,14 @@ export default class Feed extends PureComponent {
     hasMore: true,
   }
 
+  getContext = (data) => {
+    if (!data) return [];
+
+    const { me: { feed: { groups } } } = data;
+    const objects = flatten(map(groups, group => group.objects));
+    return filter(objects, object => (object.__typename !== 'Channel' && object.__typename !== 'Group' && object.__typename !== 'User'));
```
Comment (dzucconi) [inline react/components/Feed/index.js]: https://github.com/aredotna/ervell/blob/master/react/components/Cell/components/Konnectable/index.js#L99 also?

File: react/components/Feed/index.js [hunk 3]
```diff
@@ -63,10 +73,11 @@ export default class Feed extends PureComponent {
           return (
             <InfiniteScroll
               pageStart={1}
-              threshhold={500}
+              threshhold={1000}
               loader={<BlocksLoadingIndicator key="loading" />}
               hasMore={hasMore}
               loadMore={() => {
+                console.log('LOAD MORE');
```
Comment (dzucconi) [inline react/components/Feed/index.js]: 👋

File: react/util/updateMediatorBlocks.js (hunk 1 above)
Comment (broskoski) [reply react/util/updateMediatorBlocks.js]: I can remove it, but eventually the next page will load and `mediator.blocks` will need to be updated.

File: react/components/Feed/index.js (hunk 2 above)
Comment (broskoski) [reply react/components/Feed/index.js]: Groups and users are on the feed, didn't seem like something that belonged in Konnectable?

Comment (dzucconi) [reply react/components/Feed/index.js]: I don't know I don't remember any of how I got it working. Realistically it doesn't belong in either place

**Pull request merged**